release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4372,3 +4372,34 @@ $ oc adm release info 4.13.50 --pullspecs
 [id="ocp-4-13-50-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.13.51
+[id="ocp-4-13-51"]
+=== RHSA-2024:6811 - {product-title} 4.13.51 bug fix and security updates
+
+Issued: 25 September 2024
+
+{product-title} release 4.13.51, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6811[RHSA-2024:6811] advisory. TThe RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:6813[RHBA-2024:6813] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.51 --pullspecs
+----
+
+[id="ocp-4-13-51-enhancements"]
+==== Enhancements
+
+* CENTOS 8 has recently ended its lifecycle. This release updates the CENTOS 8 references to CENTOS 9. (link:https://issues.redhat.com/browse/OCPBUGS-39160[*OCPBUGS-39160*])
+
+[id="ocp-4-13-51-bug-fixes"]
+==== Bug fixes
+
+* Previously, `openvswitch` could function by using older configurations after a cluster upgrade operations. These older configurations might have caused issues for networking components that interact with `openvswitch`. With this release, `openvswitch` is restarted after an upgrade operation so that `openvswitch`  is forced to receive the latest configurations. (link:https://issues.redhat.com/browse/OCPBUGS-41723[*OCPBUGS-41723*])
```

Comment:
Typo + white space, but it also seems like it's mostly referred to across the repo generically as "Open vSwitch (OVS)":
```suggestion
* Previously, Open vSwitch (OVS) could function by using older configurations after cluster upgrade operations. These older configurations might have caused issues for networking components that interact with OVS. With this release, OVS is restarted after an upgrade operation so that OVS is forced to receive the latest configurations. (link:https://issues.redhat.com/browse/OCPBUGS-41723[*OCPBUGS-41723*])
```